tests/i2c_device/src/test.xc

```diff
@@ -1,11 +1,11 @@
-// Copyright 2016-2021 XMOS LIMITED.
+// Copyright 2016-2024 XMOS LIMITED.
 // This Software is subject to the terms of the XMOS Public Licence: Version 1.
 #include <xs1.h>
 #include <stdio.h>
 #include <stdlib.h>
 #include "control.h"
-#include "control_transport.h"
-#include "control_host_support.h"
+#include "control_shared.h"
+#include "../../../lib_device_control/host/control_host_support.h"
```

Comment:
What has prevented the build system from including the path to this header file in the list of include directories?

Reply:
I wanted to avoid updating the module_build_info, as this is used only in XC, so I thought the easiest way was to use the relative path. I could have added the include path in the XCC_FLAGS too.
Now I realized that there is an I2C side for XC too, so I put the include path in the module_build_info.